core-it-suite/src/test/java/org/apache/maven/it/MavenITmng6720FailFastTest.java

```diff
@@ -36,7 +34,7 @@
  * <a href="https://issues.apache.org/jira/browse/MNG-6720">MNG-6720</a>.
  *
  */
-public class MavenITmng6720FailFastTest
+class MavenITmng6720FailFastTest
```

Comment:
Is the rest of our tests also package private?

Reply:
No yet, it is Junit5 convention ,so step by step